services/web/client/source/class/osparc/ui/toolbar/FetchButton.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * oSPARC - The SIMCORE frontend - https://osparc.io
+ * Copyright: 2020 IT'IS Foundation - https://itis.swiss
+ * License: MIT - https://opensource.org/licenses/MIT
+ * Authors: Ignacio Pascual (ignapas)
+ */
+
+/**
+ * A toolbar button that serves to fetch or load some data from the server. To indicate that some processing is being done, and
+ * that the user has to wait, a rotating special icon is shown meanwhile.
+ */
+qx.Class.define("osparc.ui.toolbar.FetchButton", {
```

Comment:
Can this class share/inherit part of the code of osparc.ui.form.FetchButton?

Reply:
done